config/apparmor/abstractions/start-container

```diff
@@ -41,3 +41,4 @@
 
   change_profile -> lxc-*,
   change_profile -> unconfined,
+  change_profile -> :lxc-*:unconfined,
```

Comment:
Is that the right syntax?

For LXD we use:

    change_profile -> :lxc-*://*